pom.xml

```diff
@@ -142,6 +142,7 @@
 		<dependency>
 			<groupId>sc.fiji</groupId>
 			<artifactId>spim_data</artifactId>
+			<version>2.0.1</version>
```

Comment:
No, I don't know how this happened. I thought I was moving the version forward actually...